lmctl/project/handlers/etsi_ns/etsi_ns_src.py

```diff
@@ -0,0 +1,176 @@
+import os, json
+import lmctl.project.handlers.assembly as assembly_api
+import lmctl.project.handlers.interface as handlers_api
+import lmctl.project.mutate.behaviour as behaviour_mutations
+import lmctl.utils.descriptors as descriptors
+import lmctl.files as files
+from .etsi_ns_content import EtsiNsPkgContentTree
+
+class EtsiNsSourceTree(assembly_api.AssemblySourceTree):
+    def __init__(self, root_path=None):
+        super().__init__(root_path)
+
+    MANIFEST_FILE = 'MRF.mf'
+    FILES_DIRECTORY = 'Files'
+    DEFINITIONS_DIR = 'Definitions'
+    TEST_CONFIG_DIRECTORY = 'Files/Tests/Configurations'
```

Comment:
To ensure lmctl can work on many platforms it's best we don't put forward slashes in path strings and instead use something like os.path.join to combine directories into a path.